CHANGELOG.md

```diff
@@ -8,21 +8,35 @@ Packaged by Tomaz Muraus <tomaz@scalyr.com> on Dec 31, 2020 14:00 -0800
 --->
 
 Improvements:
-* Add new ``tcp_request_parser`` and ``tcp_message_delimiter`` config option. Valid values for ``tcp_request_parser`` include ``default`` and ``batch``. New TCP recv batch oriented request parser is much more efficient than the default one and should be a preferred choice in most situations. For backward compatibility reasons, the default parsed hasn't been changed yet.
+* Add new ``tcp_request_parser`` and ``tcp_message_delimiter`` config option. Valid values for ``tcp_request_parser``
+include ``default`` and ``batch``. New TCP recv batch oriented request parser is much more efficient than the default
+one and should be a preferred choice in most situations.  For backward compatibility reasons, the default parser hasn't
+been changed yet.
 
 Misc:
 * Update docker monitor so we don't log some non-fatal errors under warning log level when consuming logs using Docker API.
 * Add support for ``compression_type: none`` config option which completely disables compression for outgoing requests. Right now one of the main bottle necks in the high volume scenarios in the agent is compression operation. Disabling it can, in some scenarios, lead to large increase to the overall throughput (up to 2x). Disabling the compression will in most cases result in larger data egress traffic which may incur additional charges on your infrastructure provider so this option should never be set to ``none`` unless explicitly advised by the technical support.
 * Linux system metrics monitor has been updated to also ignore ``/var/lib/docker/*`` and ``/snap/*`` mount points by default. Capturing metrics for those mount points usually offers no additional insight to the end user. For information on how to change the ignore list via configuration option, please see [RELEASE_NOTES](https://github.com/scalyr/scalyr-agent-2/blob/master/RELEASE_NOTES.md).
 
-## 2.1.16 "Lasso" - December 23, 2020
+## 2.1.16 "Lasso" - January 13, 2021
 
 <!---
-Packaged by Tomaz Muraus <tomaz@scalyr.com> on Dec 23, 2020 14:00 -0800
+Packaged by Arthur Kamalov <arthur@scalyr.com> on Jan 13, 2021 14:00 -0800
 --->
 
+Features:
+* Add copy truncate log rotation support. Use the ``enable_copy_truncate_log_rotation_support`` config option to enable it.
```

Comment:
It is actually enabled by default, so folks shouldn't have to do anything.  I would say something like:
```
Add copy truncate log rotation support.  This is enabled by default.  It does not support copy truncate with
compression unless the `delaycompress` option is used.  This feature can be disabled by setting `enable_copy_truncate_log_rotation_support` to false.
```

Reply:
Oh, thanks for the correction.